app/models/beta/search/open_search_result.rb

```diff
@@ -26,7 +28,7 @@ def build_hit(hit_result)
           hit = Hashie::TariffMash.new
 
           hit.score = hit_result._score
-          hit.goods_nomenclature_class = hit_result._source.goods_nomenclature_class
+          hit.goods_nomenclature_class = ActiveSupport::StringInquirer.new(hit_result._source.goods_nomenclature_class)
```

Comment:
Are you using StringInquirer just to avoid to do 
hit.goods_nomenclature_class == 'chapter' ?
I don't know, I prefer using just plain ruby when the gain is not much,
but I guess it's personal style

Reply:
I basically added it as I found it more expressive/could see uses for it later :shrug: . I guess we do do this in a couple of places in the backend (quota search and in some tests)